tensorflow/core/platform/posix/env_time.cc

```diff
@@ -18,6 +18,23 @@ limitations under the License.
 
 #include "tensorflow/core/platform/env_time.h"
 
+// Slightly pruned version of https://gist.github.com/alfwatt/3588c5aa1f7a1ef7a3bb
+// Copyright (c) 2015-2018 Alf Watt - Open Source - https://opensource.org/licenses/MIT
```

Comment:
You cannot include copyrighted code in TensorFlow.

Reply:
libzmq uses the exact same function and I was told at https://github.com/tensorflow/tensorflow/issues/22636#issuecomment-426488627 that the wheels were built with the implementation from libzmq. 

Should I just update my PR to say that it has been taken from libzmq instead?
https://github.com/zeromq/libzmq/blob/cfc9d5f5198e611d8d3ad4482d57cf6bb5b6bf59/src/clock.cpp#L60-L70